whatsappium.js

```diff
@@ -8,32 +8,31 @@ function dispatchClick (item) {
 
 
 function makeActive (item) {
-  const chat = item.querySelector('.chat');
+  const chat = item.querySelector('._2EXPL');
   if (chat) {
     dispatchClick(chat);
   }
 }
 
 
 function isItemActive (item) {
-    const chat = item.querySelector('.chat');
-    return chat && chat.classList.contains('active');
+    const chat = item.querySelector('._2EXPL');
+    return chat && chat.classList.contains('_1f1zm');
```

Comment:
hmmm it seems like these are derived CSS classes (maybe they are using something like SASS). Is there anything else we can use that is a bit more human readable and less likely to change over time?

Reply:
Yes, it is annoying, and will likely change if they ship a rebuilt version.  Before I found this work I had a quick go at doing it myself and there is nothing obvious to identify the right elements.  We could do it with expressions like "div[dir=auto] > div > etc.... > div" but they would likely be fragile too.

Reply:
For reference, this change was made on 6th October 2018 https://github.com/c-samuel/rambox-whatsapp-shortcuts-telegram-theme/commit/4d8394e558c41b1b59e77e108f135849d1d150db and it still works ... for now.